<header class="banner"><div class="banner">
  <div class="logos">
    <div><img title="Julia logo" src="../figures/julia-logo-color.svg" alt="Julia animated logo" width=35 /></div>
    <div>◌</div>
    <div><img title="Powered by MathJax" src="https://www.mathjax.org/badge/mj-logo.svg"
              alt="Powered by MathJax" width=65 />
    </div>
  </div>  <!-- logos -->
  <div class="cite_textbook ms9">4.034 <span class="authors"></span></div>
</div></header>

<footer class="navbar navbar-default navbar-fixed-bottom mcs-psy-nav" >
  <div class="container-fluid text-right h4">
    <em>ψᵟ</em>
  </div>
</footer>





<div class="mcs-bs-callout mcs-bs-callout-info">

#### work time: 20 minutes

</div>

## Given
Steam enters the first-stage turbine shown in the figure at 40 bar and 500 °C with a mass flow rate of 40,000 kg/hr. Steam exits the first-stage turbine at 20 bar and 400 °C. The steam is then reheated at constant pressure to 500 °C before entering the second-stage turbine. Steam leaves the second stage as saturated vapor at 0.6 bar. Assume steady state operation and ignore stray heat transfer and kinetic and potential energy effects. 

<img style="width: 60%;" src="../figures/EAT_14908854309950_8181991609077116.png"  />



## Find
1. Volumetric flow rate of the steam at the inlet to the first-stage turbine, in m³/min,   
1. Rate of heat transfer to the steam flowing through the reheater, in kW, and   
1. Total power produced by the two stages of the turbine, in kW.   

### Get Organized   

List your assumptions:   
1. Steady state operation  
2. No heat losses from any components  
3. No kinetic and potential energy effects.

Let's determine what information we need to find in order to answer the questions.  

1. **Volumetric flow rate of the steam at the inlet to the first-stage turbine**

  We need $\dot V_1.$ We are given $\dot m.$ We know 
  
  \begin{equation}\dot V_1 = \dot m v_1 \label{eq:volflow} \end{equation}
  
  <u>We'll resolve ⓵ to get $v_1.$</u>
  
  ----

1. **The rate of heat transfer to the steam flowing through the reheater**

  We apply the energy equation to the reheater. _Remember, the steam is flowing, so it carries internal energy and flow energy in/out of the system._  

  \begin{array}{ccccl}
   \dot E_{in} &-& \dot E_{out} &=& \color{red}\cancelto{\text{0: steady state}}{\color{black}\frac {\rm d}{\rm dt}E_{sys}}  \\[6pt] 
   \dot Q_\mathit{reheater,in} + \dot m \left( u_2 + P_2v_2 \right) &-& \dot m \left( u_3 + P_3v_3 \right) &=& 0  \\[6pt]{}
  \end{array}

  \begin{gather} \label{eq:qreheater}
   \dot Q_\mathit{reheater,in} = \dot m \left( h_3 - h_2 \right)  \\[6pt]{}\notag
  \end{gather}

  <u>We'll resolve ⓶ and ⓷ to get $ h_2, h_3.$</u>
  
  ----

1. **The total power produced by the two stages of the turbine**

  We apply the energy equation to each turbine and sum the results.  

  $$\dot W_{total,out} = \dot W_\textit{first stage, out} + \dot W_\textit{second stage, out}$$  
  
  We'll start with first stage turbine. _Again, the steam is flowing, so it carries internal energy and flow energy in/out of the system._   

  \begin{array}{ccccl}
   \dot E_{in} &-& \dot E_{out} &=& \color{red}\cancelto{\rlap{\text{0: steady state}}}{\color{black}\frac {\rm d}{\rm dt}E_{sys}} \\ 
   \dot m \left( u_1 + P_1v_1 \right) - \dot W_\textit{first stage, out} &-& \dot m \left( u_2 + P_2v_2 \right) &=& 0 
 \end{array}

  \begin{equation*}
   \dot W_\textit{first stage, out} = \dot m \left( h_1 - h_2 \right)   
  \end{equation*}

  Likewise, 
  
  \begin{equation*}
   \dot W_\textit{second stage, out} = \dot m \left( h_3 - h_4 \right)  \\[15pt]
  \end{equation*}

  Finally, 
  
  \begin{equation}
   \dot W_{total,out} \label{eq:totalpower}
       = \dot W_\textit{first stage, out} + \dot W_\textit{second stage, out} 
       = \dot m \left( h_1 - h_2 + h_3 - h_4 \right)
  \end{equation}  


#### Overall, the properties we need are $$\bf v_1,\ h_1,\ h_2,\ h_3,\ h_4.$$



### Attack 

 Today, we'll use Julia, with _Unitful.jl_, a package for calculation with physical units.

### Volumetric flow rate of the steam at the inlet to the first-stage turbine** 

  From \ref{eq:volflow}, $\dot V_1 = \dot m v_1$   

  $\left.
  \begin{aligned}
  P_1 &= \pu{40 bar} \\
  T_1 &= \pu{500°C} 
  \end{aligned}
  \right\}   \rightarrow
  \left.\begin{aligned}
  v_1 &= \pu{0.08643 m^3/kg} \\
  h_1 &= \pu{3445.3 kJ/kg}
  \end{aligned}
  \right.$  

_We get $h_1$ while we're in the table, knowing we need it later._

In [1]:
using Unitful 

ṁ = 40000u"kg/hr"      # given
h₁ = 3445.3u"kJ/kg"    # table 
v₁ = 0.08643u"m^3/kg"  # table

V̇₁ = ṁ * v₁ |> u"m^3/minute"; @show V̇₁;

V̇₁ = 57.620000000000005 m^3 minute^-1


----

### The rate of heat transfer to the steam flowing through the reheater**   

  From eqn \ref{eq:qreheater}, $\dot Q_\mathit{reheater,in} = \dot m \left( h_3 - h_2 \right) $

  $\quad \left.
  \begin{alignat*}{1}
  P_3 &= 20\ \textsf{bar} \\
  T_3 &= 500\ \textsf{℃} 
  \end{alignat*}
  \right\}
  \rightarrow
   h_3 = 3467.6\, \mathrm{kJ/kg}
  $
  $\quad \left.
  \begin{alignat*}{1}
  P_2 &= 20\ \textsf{bar} \\
  T_2 &= 400\ \textsf{℃} 
  \end{alignat*}
  \right\}
  \rightarrow
   h_2 = 3247.6\, \mathrm{kJ/kg}
  $ 

In [2]:
h₃ = 3467.6u"kJ/kg"  # table 
h₂ = 3247.6u"kJ/kg"  # table 

Q̇_reheater_in = ṁ * (h₃ - h₂) |> u"kW"; @show Q̇_reheater_in;

Q̇_reheater_in = 2444.4444444444443 kW


----

### **The total power produced by the two stages of the turbine**

  From eqn \ref{eq:totalpower}, $\dot W_{total,out} = \dot m \left( h_1 - h_2 \right) + \dot m \left( h_3 - h_4 \right) $

  $\left.
  \begin{array}{l}
  P_4 = 0.6\ \textsf{bar} \\
  \text{saturated vapor}
  \end{array}
  \right\}
  \rightarrow
   h_4 = 2653.5\, \mathrm{kJ/kg}
  $

In [3]:
h₄ = 2653.5u"kJ/kg"  # table 

Ẇ_total_out = ṁ * (h₁ - h₂ + h₃ - h₄) |> u"kW"; @show Ẇ_total_out;

Ẇ_total_out = 11242.222222222224 kW


<div class="mcs-bs-callout mcs-bs-callout-success extra">

#### Finis. Now summarize.

1. Volumetric flow rate of the steam at the inlet to the first-stage turbine, in m³/min,   
  
  $$ \dot V_1 = \pu{57.62 m^3/min}$$ <br>
  
1. Rate of heat transfer to the steam flowing through the reheater, in kW, and   
  
  $$\dot Q_\mathit{reheater,in} = \pu{2444.44 kW}$$ <br>
  
1. Total power produced by the two stages of the turbine, in kW.  
  
  $$ \dot W_{total,out} = \pu{11242.22 kW} $$


  
</div>






<div class="tricks"><div class="tricks"></div></div>

<div class="mcs-bs-callout mcs-bs-callout-info ">



#### Here's another approach to calculating the total power output.
 
The First Law is the First Law, which is the First Law of Thermodynamics. It's always valid. You can apply the First Law to the entire system:

  <img style="width: 60%;" src="../figures/ms9.4.034-bigsystem.png"  />

\begin{array}{c}
 \dot E_{in} &-& \dot E_{out} &=& \color{red}\cancelto{\rlap{\text{0: steady state}}}{\color{black}\frac {\rm d}{\rm dt}E_{sys}}  \\[6pt] 
 \dot m \left( u_1 + P_1v_1 \right) + \dot Q_\pu{reheater,in}  &-& \dot m \left( u_4 + P_4v_4 \right) - \dot Q_\pu{total,out} &=& 0  \\[6pt]{}
\end{array}


\begin{gather*} 
 \bbox[5px,border:1px solid #18a2b8]{ \dot Q_\pu{total,out} = \dot m \left( h_1 - h_4 \right) + \dot Q_\pu{reheater,in}  = 0 }
\end{gather*}


Now, recalling eqn \ref{eq:qreheater},

  \begin{gather*} 
   \dot Q_\mathit{reheater,in} = \dot m \left( h_3 - h_2 \right),  \\[6pt]{}\notag
  \end{gather*}
  
and
  
\begin{align*} 
  \dot Q_\pu{total,out} 
  &= \dot m \left( h_1 - h_4 \right) + \dot m \left( h_3 - h_2 \right) \\[6pt] 
  &= \dot m \left( h_1 - h_2 + h_3 - h_4 \right), \\{}
\end{align*} 

which is exactly what we got in eqn \ref{eq:totalpower}... but it better be, right?? After all, the system must obey the First Law, however you define that system. Full stop. 

Simple examination shows us that **in this case**, we have to resolve all the same states and perform all the same calculations, no matter which approach we chose. However, there are many cases where wrapping your system arond everything is the best approach.

</div>

<footer class="psy"><div class="psy">ψᵟ</div></footer>